_opensearch/query-dsl/geo-and-shape.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: default
+title: Geographic and shape queries
+parent: Query DSL
+nav_order: 70
+---
+
+# Geographic and shape queries
+
+You can search multiple dimensions with the geographic and shape query types. You can create queries to search two-dimensional geometries that map out cartesian data such as `point` fields that support x and y pairs and `shape` fields that support points, lines, circles and polygon shapes.
+
+## Geographic queries
```

Comment:
Please separate geographic queries from Cartesian ones. Everything that starts with `geo` should go into the geographic queries section, and everything that starts with `xy` should go into Cartesian (xy) section.